jobs/build/advisories/Jenkinsfile

```diff
@@ -250,56 +177,12 @@ node {
                     echo "out: ${out}"
                 }
             }
-            stage("add placeholder bugs to advisories") {
-                kind = params.VERSION.startsWith("3") ? "image" : "rpm"
-
-                cmd = "-g openshift-${params.VERSION} "
-                cmd+= "create-placeholder "
-                cmd+= "--kind ${kind} "
-                cmd+= "--use-default-advisory ${kind}"
-
-                echo "elliott cmd: ${cmd}"
-                if (params.DRY_RUN) {
-                    out = "DRY RUN mode, command did not run"
-                } else {
-                    out = buildlib.elliott(cmd, [capture: true])
-                }
-                echo "out: ${out}"
-
-                // Extras placeholder
-                if (params.VERSION.startsWith("4")) {
-                    cmd = """
-                        -g openshift-${params.VERSION}
-                        create-placeholder
-                        --kind extras
-                        --use-default-advisory extras
-                    """
-
-                    echo "elliott cmd: ${cmd}"
-                    if (params.DRY_RUN) {
-                        out = "DRY RUN mode, command did not run"
-                    } else {
-                        out = buildlib.elliott(cmd, [capture: true])
-                    }
-                    echo "out: ${out}"
-                }
 
-                // OLM Operators metadata placeholder
-                if (params.VERSION.startsWith("4")) {
-                    cmd = """
-                        -g openshift-${params.VERSION}
-                        create-placeholder
-                        --kind metadata
-                        --use-default-advisory metadata
-                    """
-
-                    echo "elliott cmd: ${cmd}"
-                    if (params.DRY_RUN) {
-                        out = "DRY RUN mode, command did not run"
-                    } else {
-                        out = buildlib.elliott(cmd, [capture: true])
-                    }
-                    echo "out: ${out}"
+            stage("add placeholder bugs to advisories") {
+                lib.ADVISORIES.each {
+                    if (it.key == "rpm" && params.VERSION.startsWith("3.")) { return }
+                    if (it.key == "image" && params.VERSION.startsWith("4.")) { return }
+                    lib.create_placeholder(it.key)
```

Comment:
actually RHSAs will never need placeholders - by nature they must have bugs. 
we could probably invert this logic a bit:
```suggestion
                    if (it.key == "rpm" && params.VERSION.startsWith("4.")) { lib.create_placeholder(it.key) }
                    if (it.key == "image" && params.VERSION.startsWith("3.")) { lib.create_placeholder(it.key) }
                    if (it.key == "extras") { lib.create_placeholder(it.key) }
```